packages/fluentui/react-bindings/test/hooks/useTriggerElement-test.tsx

```diff
@@ -24,7 +24,7 @@ describe('useTriggerElement', () => {
   });
 
   it('"trigger" can be null', () => {
```

Comment:
```suggestion
  it('"trigger" can be undefined', () => {
```

Reply:
👍 will update this shortly--I'd considered this change before but I wasn't sure if null was special and specifically needed to be supported for some reason.